integration_tests/example_05_browser_node.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
Could you add this test to the Makefile? I'd like to see the CI failing if this is not yet passing.

Reply:
(please ignore the k8s test suite, it is flaky)

Reply:
I was planning to do only once my test is actually running. For now it is blocked on sending over a WS to the sync service. Planning to get that working this week.

Reply:
I already opened the PR more to sync on the direction and what is still planned to be done as part of this PR

Reply:
@laurentsenta added the PR for `@testground/sdk` as well in this PR description: <https://github.com/testground/sdk-js/pull/25>